esiclient/tests/v1/test_node_network.py

```diff
@@ -317,6 +317,38 @@ def test_take_action_port(self, mock_gfnifp):
             assert_called_once_with('node1', self.neutron_port.id)
         mock_gfnifp.assert_called_once
 
+    @mock.patch('esiclient.utils.get_full_network_info_from_port',
+                return_value=(["test_network"], ["2.2.2.2"]),
+                autospec=True)
+    def test_take_action_port_and_mac_address(self, mock_gfnifp):
+        self.app.client_manager.baremetal.node.get.\
+            return_value = self.node
+        self.app.client_manager.baremetal.port.get_by_address.\
+            return_value = self.port2
+
+        arglist = ['node1', '--port', 'node1',
+                   '--mac-address', 'bb:bb:bb:bb:bb:bb']
```

Comment:
Why does  'node1' appear twice?

Reply:
Oh, because I cleverly named the test node and the test port 'node1'. I'll update the port name so it's clearer - nice catch!